ci/do_ci.ps1

```diff
@@ -116,6 +143,28 @@ switch ($action) {
       exit $exit
     }
   }
+  "cmake.exporter.otprotocol.dll.test" {
```

Comment:
These 2 tests try to mirror `cmake.test` and `cmake.exporter.otprotocol.test`. Or do you mean the latter is a subset of the first one? I assumed no because OTLP is not turned on by default in the first test. Let me know if this is wrong.

I think it is a good to add test for `WITH_STL=ON`. Will see how to add that.